### PR TITLE
Document how to run the test suite

### PR DESCRIPTION
**Pull request by jish**
The contributing guide should explain how to run the test suite.

### CODE REVIEW

**Comment by abritinthebay** [comment]
It would be `npm test` surely? No dependency on Yarn and npm is installed with Node. Why add documentation for an inferior and additional dependency? Seems strange.

**Comment by jish** [comment]
I have updated the documentation to use `npm test` instead.

**Comment by jish** [comment]
According to https://github.com/date-fns/date-fns/issues/592 and https://github.com/date-fns/date-fns/issues/614 `yarn test -- --single-run` should be used to run tests. I have updated this branch to reflect that. 🙂

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Please follow the main contributing rules, to maintain date-fns' top quality:
 
 - Write tests.
 
+- Run tests: `$ yarn test -- --single-run`.
```
Comment (tusbar) [inline CONTRIBUTING.md]: Since Yarn 1.0, `--` is not required anymore to forward options.

```
yarn test --single-run
```

**Comment by jish** [comment]
I have removed the extraneous `--`

**Pull request closed without merging**